lib/ethpy/ethpy/hyperdrive/deploy.py

```diff
@@ -229,79 +226,62 @@ def _deploy_hyperdrive_factory(
     (base_token_contract, factory_token_contract, pool_contract_address): tuple[Contract, Contract, ChecksumAddress]
         Containing the deployed base token, factory, and the pool contracts/addresses.
     """
-    # args = [name, symbol, decimals, admin_addr, isCompetitionMode]
-    args = ["Base", "BASE", 18, ADDRESS_ZERO, False]
-    base_token_contract_addr, base_token_contract = deploy_contract(
-        web3,
-        abi=abis["ERC20Mintable"],
-        bytecode=bytecodes["ERC20Mintable"],
-        deploy_account_addr=deploy_account_addr,
-        args=args,
+    erc20args = ERC20MintableContract.ConstructorArgs("Base", "BASE", 18, ADDRESS_ZERO, False)
```

Comment:
I like the pattern of commenting the solidity arg name next to each of these. It makes it easy to cross-reference against the sol file when debugging. I was doing just that yesterday. So, for example, with this line:

```python
erc20args = ERC20MintableContract.ConstructorArgs(
    "Base", # name
    "BASE", # symbol
    18, # decimals
    ADDRESS_ZERO, # admin_addr
    False # isCompetitionMode
)
```

Reply:
but now you can just hover the ConstructorArgs to see what they are:

<img width="760" alt="image" src="https://github.com/delvtech/agent0/assets/9002261/c7cd8c50-5ba8-4524-9716-47e4b0b3a594">

but I'll do a solid here and just declare like this:
<img width="767" alt="image" src="https://github.com/delvtech/agent0/assets/9002261/a4f53edc-2464-4225-9e44-153f2f6c3f81">

Reply:
oh wow that's awesome! Ok cool yeah I like using the kwargs but the argument names on hover is 100x better than commented names 🤩